src/pages/HomePage.jsx

```diff
@@ -1,9 +1,71 @@
+import { Link } from 'react-router-dom';
+
+import { Icon } from 'ui/icons/Icon';
+
 
 export function HomePage() {
     return (
-        <section>
-            <h1>Home sweet Home</h1>
-        </section >
-    )
+      <>
+        <header className='header'>
+          <nav className='home-nav'>
+            <Link to='/home' className='logo-link'>
+              <Icon name='trello' />
+              <span>Prello</span>
+            </Link>
+            <div className='btns'>
+              <button className='home-btn'>
+                Features
+                <Icon name='chevronDown' size='14px' />
+              </button>
+              <button className='home-btn'>
+                Solutions
+                <Icon name='chevronDown' size='14px' />
+              </button>
+              <button className='home-btn'>
+                plans
+                <Icon name='chevronDown' size='14px' />
+              </button>
+              <button className='home-btn'>
+                Pricing
+                <Icon name='chevronDown' size='14px' />
+              </button>
+              <button className='home-btn'>
+                Resources
+                <Icon name='chevronDown' size='14px' />
+              </button>
+            </div>
+            <Link className='nav-link' to='/boards'>
+              Go to Boards
+            </Link>
+          </nav>
+        </header>
+
+        <main className='home-main'>
+          <div className='bg'>
+            <section className='content'>
+              <div className='left'>
+                <h1>
+                  Prello brings all your tasks, teammates, and tools together
+                </h1>
+                <p>
+                  Keep everything in the same place even if your team isn’t.
+                </p>
+                <form className='form'>
+                  <input
+                    className='email-input'
+                    type='email'
+                    placeholder='Enter your email'
+                  />
+                  <button className='email-button'>Sign up it's free!</button>
+                </form>
+              </div>
+              <pciture className='right'>
+                <img src='./src/ui/Icons/png-icons/HomeImage.png' />
```

Comment:
Move the png to assets folder inside homepage folder. 
inside pages folder create homepage folder and move there the homepage.jsx, also create assets folder and add there the images. REMOVE THEM FROM ICONS folder